mne/epochs.py

```diff
@@ -2700,6 +2700,14 @@ def _concatenate_epochs(epochs_list, with_data=True):
         if epochs.baseline != baseline:
             raise ValueError('Baseline must be same for all epochs')
 
+        # compare event_id
+        common_keys = list(set(event_id.keys()).intersection(
```

Comment:
actually epochs.event_id should always be dict, right? With `epochs = mne.Epochs(..., event_id=1)` you get `epochs.event_id == {'1': 1}`.